pandar_pointcloud_at128/src/lib/CMakeLists.txt

```diff
@@ -1,13 +1,15 @@
 add_library(pandar_rawdata_at128 rawdata.cc calibration.cc)
+add_dependencies(pandar_rawdata_at128 ${${PROJECT_NAME}_EXPORTED_TARGETS})
 target_link_libraries(pandar_rawdata_at128 
                       ${catkin_LIBRARIES}
                       ${YAML_CPP_LIBRARIES})
 install(TARGETS pandar_rawdata_at128
         RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
         ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
         LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
-set_target_properties(pandar_rawdata_at128 PROPERTIES
-	COMPILE_FLAGS -std=c++11)
+# set_target_properties(pandar_rawdata_at128 PROPERTIES
+# 	COMPILE_FLAGS -std=c++11)
+set_property(TARGET pandar_rawdata_at128 PROPERTY CXX_STANDARD 14)
```

Comment:
Why was this changed? And do we need to keep it commented out or can we delete the old `set_target_properties`?

Reply:
This change was to prevent pandar_raw_data not being built properly and we can delete the commented lines.

Reply:
We can remove it.